src/components/include/utils/macro.h

```diff
@@ -159,12 +159,15 @@
 
 #ifdef BUILD_TESTS
 #define FRIEND_TEST(test_case_name, test_name) \
-  \
-friend class test_case_name##_##test_name##_Test
+                                               \
+  friend class test_case_name##_##test_name##_Test
 #endif
 
 #if defined(QT_PORT)
-#define PLATFORM_INIT(argc, argv) QCoreApplication application(argc, argv)
+#define PLATFORM_INIT(argc, argv)                    \
+  QCoreApplication application(argc, argv);          \
+  QThreadPool* pool = QThreadPool::globalInstance(); \
+  pool->setMaxThreadCount(100)
```

Comment:
1. Move the count of threads to config file.
2. Clarify requirements for this and other platforms.
3. Could we hide this thread initialization inside of ``utils::threads``? It will be more appropriate.